docs/documentation/file-formats/overview.md

```diff
@@ -0,0 +1,17 @@
+# File formats
+
+!!!Warning
+    **This section is under construction and should not be considered as accurate yet.**
+
+Here you can find all the documentation related to specific file formats used by our algorithms.
+
+<div class="grid cards" markdown>
+
+-   **JSON configuration files**
```

Comment:
Note: If Json will be the sole config file format, then this overview page can be removed entirely.

Reply:
No, it will contain more than one format. There is only one because others have not been defined yet.